src/ng/parse.js

```diff
@@ -1698,12 +1698,14 @@ function $ParseProvider() {
   var cacheExpensive = createMap();
 
   this.$get = ['$filter', '$sniffer', function($filter, $sniffer) {
+    var csp = $sniffer.csp && !$sniffer.unsafeEval;
```

Comment:
the point is that CSP headers could turn off things like unsafe-inline, but leave unsafe-eval enabled